xml/System.Threading.Tasks/TaskFactory`1.xml

```diff
@@ -596,7 +596,7 @@
       <Docs>
         <param name="tasks">The array of tasks from which to continue.</param>
         <param name="continuationFunction">The function delegate to execute asynchronously when all tasks in the <paramref name="tasks" /> array have completed.</param>
-        <param name="continuationOptions">One of the enumeration values that controls the behavior of the created continuation task. The NotOn* or OnlyOn* values are not valid.</param>
```

Comment:
The strings `NotOn* or OnlyOn*` and `NotOn* and OnlyOn*` are found in a number of other overloads with a TaskContinuationOptions parameter. Do you want to change those as well?

Reply:
You mean in other types @rpetrusha?

Reply:
No. At least I thought that there were additional occurrences in this type.

Reply:
I just fixed those.